src/grammar/test/case-34.tact

```diff
@@ -0,0 +1,4 @@
+struct A {
```

Comment:
this feature does not apply to struct/message declarations, only to their instance constructions:

```
receive("foo") {
    let to: Address = newAddress(...);
    let value: Int = 0;
    let mode: Int = SendBounceIfActionFail;

    send(SendParameters {
        to,
        value,
        mode,
    }
}
```

Reply:
Yep, sorry about that - I hadn't finished yet the unit tests, I should have left a note in the draft PR.